packages/playground/src/components/pull-request/pull-request-system-label-item.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import PullRequestTimelineItem from './pull-request-timeline-item'
+import { Avatar, AvatarFallback, AvatarImage, Icon, Text } from '@harnessio/canary'
+import { getInitials } from '../../utils/utils'
+import { GeneralPayload, LabelActivity, PayloadAuthor, TypesPullReqActivity } from './interfaces'
+
+interface PullRequestSystemLabelItemProps {
+  payload?: TypesPullReqActivity | undefined
+  isLast: boolean
+  avatarUrl: string
+}
+const PullRequestSystemLabelItem: React.FC<PullRequestSystemLabelItemProps> = ({ payload, isLast, avatarUrl }) => {
+  return (
+    <PullRequestTimelineItem
+      key={payload?.id} // Consider using a unique ID if available
+      header={[
+        {
+          avatar: (
+            <Avatar className="w-6 h-6 rounded-full p-0">
+              <AvatarImage src={avatarUrl} />
+
+              <AvatarFallback>
+                <Text size={1} color="tertiaryBackground">
+                  {/* TODO: fix fallback string */}
+                  {getInitials((payload?.author as PayloadAuthor)?.display_name || '')}
```

Comment:
Isn't `payload?.author` of type `PayloadAuthor`. Any reason we need the type cast here?

Reply:
fixed